Lib/csharp/std_set.i

```diff
@@ -0,0 +1,239 @@
+/* -----------------------------------------------------------------------------
+ * std_map.i
+ *
+ * SWIG typemaps for std::set<T>
+ *
+ * The C# wrapper is made to look and feel like a C# System.Collections.Generic.HashSet<>.
```

Comment:
Wouldn't `System.Collections.Generic.SortedSet<>` be a better model to base this off?

Reply:
Add in:
```
 * Requires .NET 4 minimum.
```